### PR TITLE
Change Jobs pages content

### PR DESCRIPTION
**Pull request by bjgordon**

### CODE REVIEW

File: pages/who-we-are/corporate/jobs/information-pack.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 ---
 layout: basic
-title: Job Information Pack
+title: Before you apply
 permalink: /who-we-are/corporate/jobs/job-information-pack/
```
Comment (bjgordon) [inline pages/who-we-are/corporate/jobs/information-pack.md]: Change the permalink to reflect the new title, add a redirect from the existing permalink.

File: pages/who-we-are/corporate/jobs/jobs.md [hunk 2]
```diff
@@ -9,7 +9,13 @@ searchexcerpt: We are always on the lookout for talented developers, user resear
 ---
 We’re making Australian Government services simpler, clearer and faster. Making life a little easier for a huge number of Australians, every day.
 
-And we need smart, capable and dedicated people to help us do it. We have put together [some information](/who-we-are/corporate/jobs/job-information-pack/) to assist you in applying for vacancies with us.
+And we need smart, capable and dedicated people to help us do it. 
+
+Sound like you? We'd love to hear from you.
+
+[Before you apply](/who-we-are/corporate/jobs/job-information-pack/)
```
Comment (bjgordon) [inline pages/who-we-are/corporate/jobs/jobs.md]: The document had this link _after_ the recruiterbox div - is there a reason you've added it above?

File: pages/who-we-are/corporate/jobs/information-pack.md (hunk 1 above)
Comment (skpandey27) [inline pages/who-we-are/corporate/jobs/information-pack.md]: Modified the permalink and added redirect.

File: pages/who-we-are/corporate/jobs/jobs.md (hunk 2 above)
Comment (skpandey27) [inline pages/who-we-are/corporate/jobs/jobs.md]: IMO that would let applicants to check their eligibility before they go into details of jobs available.

Comment (bjgordon) [inline pages/who-we-are/corporate/jobs/jobs.md]: Ok no worries

File: pages/who-we-are/corporate/jobs/information-pack.md [hunk 3]
```diff
@@ -1,12 +1,15 @@
 ---
 layout: basic
-title: Job Information Pack
-permalink: /who-we-are/corporate/jobs/job-information-pack/
+title: Before you apply
+permalink: /who-we-are/corporate/jobs/before-you-apply/
+redirect_from:
+  - /job-information-pack/
```
Comment (bjgordon) [inline pages/who-we-are/corporate/jobs/information-pack.md]: The redirect should be from /who-we-are/corporate/jobs/job-information-pack/

**Comment by bjgordon** [comment]
LGTM thanks @skpandey27

**Pull request merged**